canonical/ubuntu.com
# [WD-8387] update /download/risc-v for 24.04 release

## Done

- Remove StarFive VisionFive tab from /download/risc-v
- Replace links in all tabs to download 24.04 release images

## QA

- Navigate to /download/risc-v and check if changes are according to those specified in the [copy doc](https://docs.google.com/document/d/1kkJeq9HdH3iWV35HwihcJd3CxZw9Jy21W66lJl8VbWs/edit)
- Please note that the links in `Download 24.04` button are currently broken because 24.04 release hasn't been published yet. That's why this change will go to a separate `24.04` feature branch for now. For now you can only check the correctness of the links themselves and compare them to ones specified in the copy doc.


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8387

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
